### PR TITLE
Add/turn off collisions zoom level migen 2381

### PR DESCRIPTION
**Pull request by JG1995**

### CODE REVIEW

File: src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+<mi-tab label="Web" tab-for="Web"></mi-tab>
+<mi-tab-panel id="Android">
+
+For Android, there are two ways of implementing this, and which you should use depends on your desired map behavior.
+
+### Google Maps for Android
+
+If you wish for the collision behavior to change when the maps stops moving, you should use this piece of code. This would generally be the most performance-friendly option.
+
+```java
```
Comment (matiasholst) [inline src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md:25]: This is not java code, but kotlin code

Comment (matiasholst) [reply src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md:25]: It's the same for all the android examples

File: src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+//2. Use GMSMApViewDelegate to listen to changes in one of four events mapView:didChangeCameraPosition
+
+self.mapView?.delegate = self //this is needed since the delegate will inform of event updates on the map view; we will use it below
+
+
+//the following is invoked on tap. you can place zoom checking anywhere in your code that is being updated
+func mapView(_ mapView: GMSMapView, didTapAt coordinate: CLLocationCoordinate2D) {
+    //3. do a check against the current projection level and make changes
+    //the following is the actual to be put in method that is invoked everytime there is a zoom level change
+    if ( zoomRange ~= (self.mapView?.camera.zoom)!) {
```
Comment (matiasholst) [inline src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md:106]: Does this not need an else clause to turn it off again?

File: src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+```swift
+//Define zoom range
+let minZoom : Float = 16.0
+let maxZoom : Float = 22.0
+let zoomRange = (minZoom...maxZoom)
+
+//1. Implement GMSMapViewDelegate on your view controller
+//class [YourClassNAme]: UIViewController, GMSMapViewDelegate
+
+//2. Use GMSMApViewDelegate to listen to changes in one of four events mapView:didChangeCameraPosition
```
Comment (ChristianWolf-mapspeople) [inline src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md:97]:
```suggestion
//2. Use GMSMapViewDelegate to listen to changes in one of four events mapView:didChangeCameraPosition
```

File: src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md (hunk 2 above)
Comment (ChristianWolf-mapspeople) [reply src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md:106]: I would say yes :)

File: src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+self.mapView?.delegate = self //this is needed since the delegate will inform of event updates on the map view; we will use it below
+
+
+//the following is invoked on tap. you can place zoom checking anywhere in your code that is being updated
+func mapView(_ mapView: GMSMapView, didTapAt coordinate: CLLocationCoordinate2D) {
+    //3. do a check against the current projection level and make changes
+    //the following is the actual to be put in method that is invoked everytime there is a zoom level change
+    if ( zoomRange ~= (self.mapView?.camera.zoom)!) {
+        MPMapControl.locationHideOnIconOverlapEnabled = true
+    }
```
Comment (ChristianWolf-mapspeople) [inline src/content/map/displaying-objects/turn-off-collisions-based-on-zoom.md:108]:
```suggestion
    } else {
        MPMapControl.locationHideOnIconOverlapEnabled = false
    }
```

**Review by ChristianWolf-mapspeople — CHANGES_REQUESTED** [review]
A few modifications for iOS

**Review by matiasholst — APPROVED** [review]

**Pull request merged**